source/common/http/http1/codec_impl.h

```diff
@@ -72,6 +72,7 @@ class StreamEncoderImpl : public virtual StreamEncoder,
   const Network::Address::InstanceConstSharedPtr& connectionLocalAddress() override;
 
   void isResponseToHeadRequest(bool value) { is_response_to_head_request_ = value; }
+  void isResponseToConnectRequest(bool value) { is_response_to_connect_request_ = value; }
```

Comment:
nit: I would probably do s/is/set in the method name and change the head request one also but up to you.